source/handlers/client.py

```diff
@@ -27,21 +31,37 @@ def __init__(self, sock, clients, ip, port):
         self._port = port
         self._running = True
         self._socket_lock = Lock()
-
         print '[*] %s:%d has connected to the server' % (ip, port)
 
     def run(self):
         while self._running:
             try:
                 request = self._recieve_request()
-                self._socket.send(json.dumps(request))
+                request = Request.deserialize(request)
+                self.handle_request(request.code, request.args)
+
```

Comment:
Why is this spacing here needed? Personally, I don't like spacing between a try block and an except keyword